.github/workflows/pullpreview.yml

```diff
@@ -38,3 +38,4 @@ jobs:
         AWS_ACCESS_KEY_ID: "${{ secrets.AWS_ACCESS_KEY_ID }}"
         AWS_SECRET_ACCESS_KEY: "${{ secrets.AWS_SECRET_ACCESS_KEY }}"
         AWS_REGION: eu-central-1
+        OPENPROJECT_FEATURE__STORAGES__MODULE__ACTIVE: true
```

Comment:
Hey @crohr, do you have any clue, why this doesn't enable the feature? Is that the wrong place to set an ENV variable for OpenProject that is specific for the PULL PREVIEW environments?

Reply:
@wielinde env variables for runtime need to be added here: https://github.com/opf/openproject/blob/4d3a3f09978bdcaf81f667764213d5c6068ad4e5/.github/workflows/pullpreview.yml#L20

Reply:
Oh, of cause. Thank you!